test/phx_tools_web/live/phx_tools_live_test.exs

```diff
@@ -0,0 +1,52 @@
+defmodule PhxToolsWeb.PhxToolsLive.IndexTest do
```

Comment:
Check the review comments for the old PR. This file should be named index_test.exs.

https://github.com/optimumBA/phx.tools/pull/2